### PR TITLE
Articles

### LINKED ISSUE #32: לעשות GroupBy
לעשות GroupBy לכתבות לפי הנושא

### PR DESCRIPTION
**Pull request by HadasKedem**

### CODE REVIEW

File: src/controllers/ArticleController.ts [hunk 1]
```diff
@@ -1,8 +1,8 @@
 import {AbstractController} from "./AbstractController";
-import {model} from "../models/Artifact"
+import {model} from "../models/Article"
 
 export class ArtifactController extends AbstractController {
```
Comment (guyeise5) [inline src/controllers/ArticleController.ts]: Change the class name

File: src/models/Article.ts [hunk 2]
```diff
@@ -5,10 +5,13 @@ const mongoose = require('mongoose'),
...
-const ArtifactSchema = new Schema({
-    name: String,
-    text: String,
+const ArticleSchema = new Schema({
+    ID: String,
     title: String,
+    subTitle: String,
+    content: String,
+    writer: String,
+    field: String
```
Comment (guyeise5) [inline src/models/Article.ts:14]: What is this?

**Review by guyeise5 — CHANGES_REQUESTED** [review]
More places where name 'Artifact' is used instead of 'Article'

File: src/controllers/ArticleController.ts (hunk 1 above)
Comment (ShacharOr100) [reply src/controllers/ArticleController.ts]: It is changed

File: src/models/Article.ts (hunk 2 above)
Comment (ShacharOr100) [reply src/models/Article.ts:14]: all fields required for article.

**Pull request merged**